### PR TITLE
Add Environment variables page

### LINKED ISSUE #36: Prepare a document on general environment variables in dit docs

### PR DESCRIPTION
**Pull request by 5h4k4r**
Resolves #36

### CODE REVIEW

File: kubernetes/fleet.yaml
Comment (MohamadTahir) [inline kubernetes/fleet.yaml:1]: I believe this file should be in the root dir.

File: docs/Software Development/06-Environment Variables.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+The following table lists the environment variables that are common to all DIT micro-service applications, along with their descriptions.
+
+| Environment Variable       | Description                                                               |
+| :------------------------- | :------------------------------------------------------------------------ |
+| CAS\_\_URL                 | The URI format for the connection string of database server               |
+| CAS\_\_M2M_CLIENT_ID       | The URI format for the connection string of database server               |
+| CAS\_\_M2M_CLIENT_SECRET   | The URI format for the connection string of database server               |
+| DATABASE\_\_URL            | The URI format for the connection string of database server               |
```
Comment (MohamadTahir) [inline docs/Software Development/06-Environment Variables.md:12]: add DATABASE__CONNECTION_STRING

File: docs/Software Development/06-Environment Variables.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The following table lists the environment variables that are common to all DIT micro-service applications, along with their descriptions.
+
+| Environment Variable       | Description                                                               |
+| :------------------------- | :------------------------------------------------------------------------ |
+| CAS\_\_URL                 | The URI format for the connection string of database server               |
```
Comment (MohamadTahir) [inline docs/Software Development/06-Environment Variables.md:9]: Fix the description of these environment variables

File: docs/Software Development/06-Environment Variables.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+| RABBITMQ\_\_URL            | The URL the RabbitMQ server                                               |
+| RABBITMQ\_\_HOST           | The host name or IP address of the RabbitMQ server                        |
+| RABBITMQ\_\_PORT           | The port number of the RabbitMQ server                                    |
+| RABBITMQ\_\_VHOST          | The virtual host name of the RabbitMQ server                              |
+| RABBITMQ\_\_USERNAME       | The username for accessing the RabbitMQ server                            |
+| RABBITMQ\_\_PASSWORD       | The password for accessing the RabbitMQ server                            |
+| RABBITMQ\_\_EXCHANGE_NAME  | The name of the exchange for RabbitMQ server                              |
+
+In addition to these common environment variables, some applications may have specific environment variables that are related to their functionality or dependencies. These variables are prefixed with the name of the application or service that they belong to, such as `TAX_CORPORATE_REGISTRATION_PORTAL_URL`. These variables are documented in their respective application repositories or README files.
```
Comment (MohamadTahir) [inline docs/Software Development/06-Environment Variables.md:45]: no need for this section.

File: docs/Software Development/06-Environment Variables.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+| CAS\_\_M2M_CLIENT_ID         | Client ID for machine-to-machine authentication with the CAS              |
+| CAS\_\_M2M_CLIENT_SECRET     | Client secret for machine-to-machine authentication with the CAS          |
+| DATABASE\_\_CONNECTIONSTRING | The connection string of database server                                  |
+| DATABASE\_\_URL              | The URL format for the connection string of database server               |
+| DATABASE\_\_HOST             | The host name or IP address of the database server                        |
+| DATABASE\_\_USERNAME         | The username for accessing the database                                   |
+| DATABASE\_\_PASSWORD         | The password for accessing the database                                   |
+| DATABASE\_\_NAME             | The name of the database                                                  |
+| DATABASE\_\_PORT             | The port number of the database server                                    |
+| S3\_\_REGION                 | The AWS region of the S3 bucket                                           |
```
Comment (MohamadTahir) [inline docs/Software Development/06-Environment Variables.md:19]: S3__REGION is not needed.

File: docs/Software Development/06-Environment Variables.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+The following table lists the environment variables that are common to all DIT micro-service applications, along with their descriptions.
+
+| Environment Variable         | Description                                                               |
+| :--------------------------- | :------------------------------------------------------------------------ |
+| CAS\_\_URL                   | URL for the Central Authentication Service (CAS)                          |
+| CAS\_\_M2M_CLIENT_ID         | Client ID for machine-to-machine authentication with the CAS              |
+| CAS\_\_M2M_CLIENT_SECRET     | Client secret for machine-to-machine authentication with the CAS          |
+| DATABASE\_\_CONNECTIONSTRING | The connection string of database server                                  |
```
Comment (broosk1993) [inline docs/Software Development/06-Environment Variables.md:12]: Should be DATABASE__CONNECTION_STRING

**Review by broosk1993 — CHANGES_REQUESTED** [review]

File: docs/Software Development/06-Environment Variables.md (hunk 5 above)
Comment (broosk1993) [reply docs/Software Development/06-Environment Variables.md:12]: Adds also REDIS__CONNECTION_STRING

**Review by MohamadTahir — APPROVED** [review]

**Review by broosk1993 — APPROVED** [review]

**Pull request merged**